config/default.yml

```diff
@@ -782,6 +782,12 @@ Rails/Present:
   # Convert usages of `unless blank?` to `if present?`
   UnlessBlank: true
 
+Rails/PrivateTransactionOption:
+  Description: 'Avoid use of `ActiveRecord::Base.transaction(joinable: _)`.'
```

Comment:
Some reasoning could be added to the message here.

Reply:
Fixed https://github.com/rubocop/rubocop-rails/pull/1236/commits/d8b64ad47e9b4bd4337ead872d1b8695c3601414